tests/lib/prepare.sh

```diff
@@ -624,6 +624,14 @@ prepare_ubuntu_core() {
         snap alias "$rsync_snap".rsync rsync
     fi
 
+    echo "Ensure the core snap is available"
+    # Install core to keep it as part of the snapd state
+    if is_core18_system; then
+        if ! snap list core; then
+            snap install core
```

Comment:
Some tests depend on not having core available. Could we cache it so that it is installed quickly, but not always really installed? I would also encourage to look at and unify this with spread.yaml's environment variable controlling the set of snaps cached for quick install.

Reply:
@zyga sure, I'll make that change